fpsim/empowerment.py

```diff
@@ -50,30 +53,58 @@ def get_empowerment_init_vals(ppl):
     # from 0 to 100 years old.
     n = len(ppl)
 
-    empwr_states = ['paid_employment', 'sexual_autonomy', 'decision_wages', 'decision_health']
-    empowerment = {empwr_state: np.zeros(n, dtype=fpd.person_defaults[empwr_state].dtype) for empwr_state in empwr_states}
+    empwr_states = ['paid_employment', 'sexual_autonomy', 'decision_wages', 'decision_health',
+                    'fertility_intent']
```

Comment:
True! Thank you for pointing that out @mzimmermann-IDM :nerd_face:. I had put fertility in the same bag because we'd  be using it with the empowerment functions, but it doesn't have to be.

I've disentangled fertility intent from the empowerment module and from `pars['empowerment']` -- Just need to figure out why the tests are failing.